docs/source/faq.rst

```diff
@@ -682,3 +682,30 @@ An Optuna implementation example to solve Euclid TSP is as follows:
     study.optimize(objective, n_trials=10)
     lehmer_code = study.best_params.values()
     print(decode(lehmer_code))
+
+How can I ignore duplicated samples?
+------------------------------------
+
+Optuna may sometimes suggest parameters evaluated in the past and if you would like to avoid this problem, you can try out the following workaround:
+
+.. code-block:: python
+
+    import optuna
+    from optuna.trial import TrialState
+
+
+    def objective(trial):
+        # Sample parameters
```

Comment:
By following Optna's coding convention, we need to add `.` at the end of sentence:
```suggestion
        # Sample parameters.
```
Ref https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#block-and-inline-comments-are-supposed-to-be-sentences